src/Player.ts

```diff
@@ -273,13 +273,22 @@ export class Player extends Character {
 		}
 
 		if (typeof this.room === 'string') {
-			let room = state.RoomManager.getRoom(this.room);
+			let room;
+			try {
+				room = state.RoomManager.getRoom(this.room);
+			} catch (e) {
+				Logger.error(e);
+			}
+
 			if (!room) {
 				Logger.error(
 					`ERROR: Player ${this.name} was saved to invalid room ${this.room}.`
 				);
-				room =
-					state.AreaManager.getPlaceholderArea().getRoomById('placeholder');
+
+
+				room = state.AreaManager.getPlaceholderArea().getRoomById(
```

Comment:
What do you think about having the fallback be startingRoom in ranvier.json? 

Having fallback be the placeholder could likely cease any gameplay from that point. Not a critical decision or anything though. That is the decision I have gone in my repo.

I also created a separate method in RoomManager, getRoom does the same. The new method is the same as getRoom, but returns null vs throws an error.

Reply:
I avoided that tactic (using startingRoom) because I want for players to never return to the startingRoom once they leave the tutorial area, and thought others might want the same. My slightly hacky workaround when I did need to change something about the built-in placeholder was to have a server event that gets and modifies the placeholder area/room on startup but I think a longer-term and more flexible/less opinionated  fix may be to have a `placeholderRoom` defined in ranvier.json, then fallback to this built-in placeholder if `placeholderRoom` is undefined..

Reply:
Yea, that would probably be the overall best solution, defining a placeholderRoom in ranvier.json

I made my 'startingRoom' a recall point in main town, so not really a starting point. On newChar creation I assign where they would need to go. So I'm not really using it correctly either.